.github/workflows/ci.yml

```
@@ -49,6 +49,6 @@ jobs:
       - uses: julia-actions/julia-buildpkg@v1
       - uses: julia-actions/julia-runtest@v1
       - uses: julia-actions/julia-processcoverage@v1
-      - uses: codecov/codecov-action@v4
+      - uses: codecov/codecov-action@v5
         with:
           file: lcov.info
```

```suggestion
      - uses: codecov/codecov-action@v5
        with:
          files: lcov.info
```

See https://github.com/codecov/codecov-action?tab=readme-ov-file#migration-guide